src/makie.jl

```diff
@@ -471,7 +471,54 @@ function figure_box_inset()
     lines!(inset_ax1, 1:10)
     scatter!(inset_ax2, 1:10, color = :black)
     fig
-end   
+end 
+
+function scatters_in_3D()
+    GLMakie.activate!() # hide 
+    Random.seed!(123)
+    xyz = randn(10, 3)
+    x, y, z = xyz[:,1], xyz[:,2], xyz[:,3]
+    fig = Figure(resolution = (1800,600), fontsize = 26)
+    ax1 = Axis3(fig; aspect= (1,1,1), perspectiveness = 0.5,)
```

Comment:
```suggestion
    ax1 = Axis3(fig; aspect = (1, 1, 1), perspectiveness = 0.5)
```
The comma is extraneous if I'm not mistaken. Same in the lines below that.